src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/Views/EditorTemplates/Parts/WebSearch.BingWebSearchSettings.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@model Orchard.MediaLibrary.WebSearch.Models.BingWebSearchSettingsPart
+
+<fieldset>
+    <legend>@T("Bing Web Search")</legend>
+    <div>
+        @Html.TextBoxFor(m => m.ApiKey, new { @class = "text medium"})
+        <span class="hint">@T("Your private key.")</span>
+    </div>
+
+    @T("Get a free API Key on {0}", Html.Raw("<a href=\"https://azure.microsoft.com/hu-hu/services/cognitive-services/bing-web-search-api\">https://azure.microsoft.com/hu-hu/services/cognitive-services/bing-web-search-api/</a>"))
```

Comment:
This is the Hungarian URL - I'm not sure if it redirects according to the browser culture, but we should use en-US as default.